from yandex_checkout.domain.exceptions.api_error import ApiError


class AuthorizeError(ApiError):
    pass
